setup.py

```diff
@@ -1,59 +1,3 @@
-"""
-Build and install the project.
-"""
```

Comment:
Maybe these lines need to be kept to make the `black` linter happy.

Reply:
Done in 7cff5908.